[//]: # (werk v2)
# gcp_status: Fix KeyError: me-central1

key        | value
---------- | ---
date       | 2025-01-29T10:02:48+00:00
version    | 2.5.0b1
class      | fix
edition    | cre
component  | checks
level      | 1
compatible | yes

This change affects users which have configured the rule 'Google Cloud Platform (GCP) Status'.
Previously, Checkmk did not handle incidents correctly, which had set `me-central1` as an associated location.
This was the corresponding traceback:
```
  File "/omd/sites/mon_home/lib/python3/cmk/plugins/gcp/agent_based/gcp_status.py", line 72, in parse
    data.setdefault(constants.RegionMap[location.id_], []).append(incident)
KeyError: 'me-central1'
```
With this werk, the crash is fixed.
